utils/config_vcs.py

```diff
@@ -96,7 +94,7 @@ def load_config_file():
     #Work out which supercomputer you're using
     hostname = socket.gethostname()
     # galaxy head node, galaxy and magnus job nodes, zeus job nodes, garrawarla job nodes
-    if hostname.startswith('galaxy') or hostname.startswith('nid') or hostname.startswith('z')or hostname.startswith('mwa'):
+    if hostname.startswith('galaxy') or hostname.startswith('nid') or hostname.startswith('z') or hostname.startswith('mwa'):
         comp_config = GALAXY_CONFIG
```

Comment:
Shouldn't we have a config for garrawarla?

Reply:
Well perhaps I should rename this because it is basically a Pawsey config. hostname.startswith('mwa') will be true if you're on the garrawarla cluster

Reply:
Okay I think that would be better

Reply:
Fixed